database/schema.sql

```diff
@@ -5,3 +5,110 @@ set client_min_messages to warning;
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "sneakers" (
+	"productId" serial NOT NULL,
+	"brand" TEXT NOT NULL,
+	"model" TEXT NOT NULL,
+	"sku" TEXT NOT NULL,
+	"price" TEXT NOT NULL,
+	"imageUrl" TEXT NOT NULL,
```

Comment:
I could imagine a sneaker without an image. Does this need to be `NOT NULL`?

The other tables should be checked, as well. If they are marked NOT NULL you will be required to provide a value when you insert or update, which may not be what you want (for example, is `address2` really required?).